src/Peering/Peering.Test/Peering.Test.csproj

```diff
@@ -10,12 +10,7 @@
     <RootNamespace>$(LegacyAssemblyPrefix)$(PsModuleName)$(AzTestAssemblySuffix)</RootNamespace>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
   </PropertyGroup>
-
-  <ItemGroup>
```

Comment:
Is the item group removed intentionally?

Reply:
It doesn't need to be defined again in test.scproj.